chasm/tree.go

```diff
@@ -1160,6 +1171,10 @@ func (n *Node) CloseTransaction() (NodesMutation, error) {
 		return NodesMutation{}, err
 	}
 
+	if err := n.resolveDeferredPointers(); err != nil {
```

Comment:
minor: is it possible to only run this step conditionally if we do find a deferred pointer in syncSubComponents?